pkgs/applications/misc/lxterminal/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, automake, autoconf, intltool, pkgconfig, gtk2, vte
+, libxslt, docbook_xml_dtd_412, docbook_xml_xslt, libxml2, findXMLCatalogs
+, buildSingleXMLCatalog
+}:
+
+let version = "0.3.1"; in
+
+stdenv.mkDerivation rec {
+  name = "lxterminal-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/lxde/lxterminal/archive/${version}.tar.gz";
+    sha256 = "e91f15c8a726d5c13227263476583137a2639d4799c021ca0726c9805021b54c";
+  };
+
+  configureFlags = [
+    "--enable-man"
+    "--with-xml-catalog=../catalog.xml" # Generated by buildSingleXMLCatalog
```

Comment:
Alternately, it is possible to patch the m4 file to autodetect the catalog from the variable, see for example [`gnome3.cheese`](https://github.com/NixOS/nixpkgs/blob/0e7358002679ad7f35d2bd99977b9f0e403d4de2/pkgs/desktops/gnome-3/apps/cheese/default.nix#L23-L25).

Reply:
Good to know, that makes things a lot easier.

Reply:
The only issue is, the macro is in `acinclude.m4` instead of the usual `m4/gtkdoc_jh_check_xml_catalog.m4`.

Reply:
Right, and the m4 file has different contents as well. I ended up generating [my own patch](https://github.com/velovix/nixpkgs/blob/a5880ace7fbc4359d3f0d8621fff16a0977283a1/pkgs/applications/misc/lxterminal/respect-xml-catalog-files-var.patch).